crates/freighter-api-types/src/index/request.rs

```diff
@@ -75,7 +74,32 @@ pub struct Publish {
     pub links: Option<String>,
 }
 
-#[cfg_attr(feature = "client", derive(Serialize))]
+impl Publish {
+    #[must_use]
+    pub fn empty() -> Publish {
```

Comment:
This looks like `Default` in denial?

Reply:
`Version` doesn't have a `Default` impl. I felt saying it's empty is a bit more clear than default. There is no default Publish that makes any sense or will work (unless we allow empty crate names?).